<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Login</class>
    <widget class="QDialog" name="Login">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>266</width>
                <height>259</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>管理员登录</string>
        </property>
        <property name="windowIcon">
            <iconset resource="img.qrc">
                <normaloff>:/imgs/logo_icon.png</normaloff>:/imgs/logo_icon.png
            </iconset>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color:white;</string>
        </property>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>100</x>
                    <y>10</y>
                    <width>61</width>
                    <height>61</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="pixmap">
                <pixmap resource="img.qrc">:/imgs/login_av.png</pixmap>
            </property>
            <property name="scaledContents">
                <bool>true</bool>
            </property>
        </widget>
        <widget class="QLineEdit" name="user_name">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>90</y>
                    <width>151</width>
                    <height>20</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>90</y>
                    <width>31</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>账号:</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>30</x>
                    <y>130</y>
                    <width>31</width>
                    <height>21</height>
                </rect>
            </property>
            <property name="text">
                <string>密码:</string>
            </property>
        </widget>
        <widget class="QLineEdit" name="password">
            <property name="geometry">
                <rect>
                    <x>70</x>
                    <y>130</y>
                    <width>151</width>
                    <height>20</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Arial</family>
                    <pointsize>6</pointsize>
                </font>
            </property>
            <property name="inputMask">
                <string/>
            </property>
            <property name="text">
                <string/>
            </property>
            <property name="echoMode">
                <enum>QLineEdit::Password</enum>
            </property>
            <property name="readOnly">
                <bool>false</bool>
            </property>
        </widget>
        <widget class="QPushButton" name="guest_button">
            <property name="geometry">
                <rect>
                    <x>140</x>
                    <y>180</y>
                    <width>71</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="autoFillBackground">
                <bool>false</bool>
            </property>
            <property name="text">
                <string>访客</string>
            </property>
            <property name="autoDefault">
                <bool>true</bool>
            </property>
            <property name="default">
                <bool>true</bool>
            </property>
            <property name="flat">
                <bool>true</bool>
            </property>
        </widget>
        <widget class="QPushButton" name="login_button">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>180</y>
                    <width>71</width>
                    <height>23</height>
                </rect>
            </property>
            <property name="text">
                <string>登录</string>
            </property>
            <property name="checkable">
                <bool>false</bool>
            </property>
            <property name="autoDefault">
                <bool>true</bool>
            </property>
            <property name="default">
                <bool>true</bool>
            </property>
            <property name="flat">
                <bool>true</bool>
            </property>
        </widget>
    </widget>
    <resources>
        <include location="img.qrc"/>
    </resources>
    <connections/>
</ui>
